### ISSUE TITLE
FIB building

### ISSUE THREAD

**Issue by QuintenM911** [fkk-cz/noire-public#705]
I know we talked about it around a month ago just wanted to touch base with some of the things we talked about for the FIB building. The biggest thing that it needs is an Armory and Equipment officer in the lobby that would allow us to do training events much easier and in a timely manner without having to go back and collect equipment. We had also talked about a training mode that we could access from the desk that would bring us to a separate cell away from the rest of the people of LS and allow for immediate revival. Another helpful addition to this would be for trainers and personal not participating in the event to spectate in a phantom mode moving around the building. Another thing some people show interest in is have a way to have hostile ped occupy the building in large quantities in a training mode. I know some of these suggestions will be farther out if implemented the most important thing for the building would be the armor and equipment officer to facilitate training

**Comment by No-Yes1** [comment]
I believe this was sorted a while back (unless i'm thinking of something else). Can someone confirm?

**Comment by QuintenM911** [comment]
Yes the armory and SWAT ped have been added still waiting to see if we can get the option to be respawned immediately after death so we no longer need an angle to do training
